Jenkins builds failing due to Coveralls API submission

E.g. https://amplab.cs.berkeley.edu/jenkins/job/ADAM/HADOOP_VERSION=2.7.5,SCALAVER=2.11,SPARK_VERSION=2.4.3,label=ubuntu/4367/console

```
[INFO] ------------------------------------------------------------------------
[INFO] Building ADAM_2.11 0.29.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- coveralls-maven-plugin:4.3.0:report (default-cli) @ adam-parent-spark2_2.11 ---
[INFO] Starting Coveralls job for jenkins (4367 / https://amplab.cs.berkeley.edu/jenkins/job/ADAM/HADOOP_VERSION=2.7.5,SCALAVER=2.11,SPARK_VERSION=2.4.3,label=ubuntu/4367/)
[INFO] Using repository token <secret>
[INFO] Git commit e4efc04 in master
[INFO] Writing Coveralls data to <https://amplab.cs.berkeley.edu/jenkins/job/ADAM/HADOOP_VERSION=2.7.5,SCALAVER=2.11,SPARK_VERSION=2.4.3,label=ubuntu/ws/target/coveralls.json...>
[INFO] Processing coverage report from <https://amplab.cs.berkeley.edu/jenkins/job/ADAM/HADOOP_VERSION=2.7.5,SCALAVER=2.11,SPARK_VERSION=2.4.3,label=ubuntu/ws/target/cobertura.xml>
[INFO] Successfully wrote Coveralls data in 630ms
[INFO] Gathered code coverage metrics for 145 source files with 39649 lines of code:
[INFO] - 9017 relevant lines
[INFO] - 6834 covered lines
[INFO] - 2183 missed lines
[INFO] Submitting Coveralls data to API
[ERROR] Submission failed in 892ms while processing data
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] ADAM_2.11 .......................................... FAILURE [  1.888 s]
[INFO] ADAM_2.11: Shader workaround ....................... SUCCESS [  4.669 s]
[INFO] ADAM_2.11: Avro-to-Dataset codegen utils ........... SUCCESS [  9.093 s]
[INFO] ADAM_2.11: Core .................................... SUCCESS [14:59 min]
[INFO] ADAM_2.11: APIs for Java, Python ................... SUCCESS [ 44.433 s]
[INFO] ADAM_2.11: CLI ..................................... SUCCESS [01:16 min]
[INFO] ADAM_2.11: Assembly ................................ SUCCESS [  9.553 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 17:38 min
[INFO] Finished at: 2019-08-10T00:02:06-07:00
[INFO] Final Memory: 99M/1411M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.eluder.coveralls:coveralls-maven-plugin:4.3.0:
report (default-cli) on project adam-parent-spark2_2.11: Processing of input or
output data failed: Report submission to Coveralls API failed with HTTP status 422:
Unprocessable Entity (Couldn't find a repository matching this job.) -> [Help 1]
```

--- a/README.md
+++ b/README.md
@@ -2,7 +2,6 @@ ADAM
 ====
 
 [![Build Status](https://img.shields.io/jenkins/s/https/amplab.cs.berkeley.edu/jenkins/view/Big%20Data%20Genomics/job/ADAM.svg)](https://amplab.cs.berkeley.edu/jenkins/view/Big%20Data%20Genomics/job/ADAM/)
-[![Coverage Status](https://coveralls.io/repos/github/bigdatagenomics/adam/badge.svg?branch=master)](https://coveralls.io/github/bigdatagenomics/adam?branch=master)
 [![Maven Central](https://img.shields.io/maven-central/v/org.bdgenomics.adam/adam-parent-spark2_2.11.svg?maxAge=600)](http://search.maven.org/#search%7Cga%7C1%7Corg.bdgenomics.adam)
 [![API Documentation](http://javadoc.io/badge/org.bdgenomics.adam/adam-core-spark2_2.11.svg?color=brightgreen&label=scaladoc)](http://javadoc.io/doc/org.bdgenomics.adam/adam-core-spark2_2.11)
 

--- a/scripts/jenkins-test
+++ b/scripts/jenkins-test
@@ -86,8 +86,7 @@ fi
 # if those pass, build the distribution package and the integration tests
 mvn -U \
     test \
-    -P coverage,coveralls  scoverage:report coveralls:report \
-    -DrepoToken=${COVERALLS_REPO_TOKEN} ${COVERALLS_PRB_OPTION}
+    -P coverage scoverage:report
 
 # make verbose again
 set -x -v

